homodeus_perceptions/src/proc_face_detection.cpp

```diff
@@ -223,7 +239,7 @@ int main(int argc, char **argv)
   ros::init(argc,argv,"homodeus_proc_face_detection_node");
   std::string mode;
 
-  ros::NodeHandle nh;
+  ros::NodeHandle nh("~");
```

Comment:
I talked to Matthieu Labbé and this nodehandle should not be private if we want relative paths